docs/source/api/servers_id_deliveryservices.rst

```diff
@@ -50,7 +50,7 @@ Response Structure
 ------------------
 :active:                   ``true`` if the Delivery Service is active, ``false`` otherwise
 :anonymousBlockingEnabled: ``true`` if :ref:`Anonymous Blocking <anonymous_blocking-qht>` has been configured for the Delivery Service, ``false`` otherwise
-:cacheurl:                 A setting for a deprecated feature of now-unsupported Trafficserver versions
+:cacheurl:                 Deprecated. A setting for a deprecated feature of now-unsupported Trafficserver versions
```

Comment:
You should use the `deprecated` directive. It would look like this:

```rst
:cacheurl:                 A setting for a deprecated feature of now-unsupported Trafficserver versions

    .. deprecated:: 3.0
        This field is deprecated as of Traffic Control 3.0, and will be removed in version 4.0

:ccrDnsTtl:                The Time To Live (TTL) of the DNS response for A or AAAA record queries requesting the IP 
```

Reply:
To say that is removed in 4.0 may or may not be accurate though.  Wouldn't it be better to say that's only supported through 3.0 and subject to removal in 4.0 or later?

Reply:
Presumably once it's gone for good, it's also stripped from the documentation.  Trying to keep up with when issues are slated for releases in documentation seems presumptuous.

Reply:
Oh, feel free to word it however you think is best. I was just demonstrating the syntax. The only thing that's truly important is that first line `.. deprecated:: 3.0`

Reply:
Actually, I take that back. There is one other important thing. Because this is in the API docs, you should make note of the fact that it's version 3.0 of Traffic Control that deprecates it. Since we version our API separately there could be some confusion otherwise.